mlflow/pyfunc/__init__.py

```diff
@@ -628,6 +633,35 @@ def __repr__(self):
         return yaml.safe_dump({"mlflow.pyfunc.loaded_model": info}, default_flow_style=False)
 
 
+def check_requirements_and_local_installed_mismatch(local_path):
```

Comment:
```suggestion
def _check_requirements_and_local_installed_mismatch(local_path):
```

Should this function private or not?

Reply:
```suggestion
def check_requirements_and_local_installed_mismatch(model_path):
```

Reply:
Can we add docstring?